src/plugins/postgres_table_copy_operator.py

```diff
@@ -141,16 +141,19 @@ def execute(self, context: Dict[str, Any]) -> None:  # noqa: C901
                         SELECT tablename FROM pg_tables
                         WHERE schemaname = 'public' AND tablename like %(table_name)s ESCAPE '!'
                     """,
-                    dict(table_name=f"{self.source_table_name}!_%"),
+                    {"table_name": f"{self.source_table_name}!_%"},
                 )
 
-                junction_tables = tuple(map(operator.itemgetter("tablename"), cursor.fetchall()))
+                junction_tables = cast(
+                    Tuple[str, ...],
+                    tuple(map(operator.itemgetter("tablename"), cursor.fetchall())),
+                )
                 if junction_tables:
                     self.log.info(
-                        f"Found the following junction tables: '{', '.join(junction_tables)}'."
+                        "Found the following junction tables: '%s'.", ", ".join(junction_tables)
```

Comment:
Not a big deal here, but I would just log the tuple instead of stringificatenating it. Makes bugs easier to spot.

```suggestion
                        "Found the following junction tables: '%r'", junction_tables)
```

Reply:
Ideally we use [structlog](https://www.structlog.org/en/stable/) Untli that time I'm sure I like the suggested `%r` over the `%s`. Curious what your answer will be to my [question](https://github.com/Amsterdam/dataservices-airflow/pull/284#discussion_r656204464)

Reply:
Fixed as part of the original commit. I love my rebases.